commands/prepare.py

```diff
@@ -667,13 +667,21 @@ def prepare(account, config, outputfilter):
             filtered_node['data']['node_data'] = {}
             filtered_cytoscape_json.append(filtered_node)
         cytoscape_json = filtered_cytoscape_json
-    with open("web/data.json", "w") as outfile:
+    with open(outputfile, "w") as outfile:
         json.dump(cytoscape_json, outfile, indent=4)
 
 
 def run(arguments):
     # Parse arguments
     parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--output", 
+        help="Output data file path",
+        required=False, 
```

Comment:
All `--` args are not required, so for consistency with other `add_argument` calls, maybe we should remove this line.